prov/efa/src/rdm/rxr_ep.c

```diff
@@ -2397,6 +2455,12 @@ int rxr_endpoint(struct fid_domain *domain, struct fi_info *info,
 	}
 
 	rxr_ep->cuda_api_permitted = (FI_VERSION_GE(info->fabric_attr->api_version, FI_VERSION(1, 18)));
+	rxr_ep->sendrecv_in_order_aligned_128_bytes = false;
+	rxr_ep->write_in_order_aligned_128_bytes = false;
```

Comment:
Are these values uninitialized if API version is <1.18, or did I miss that?

Reply:
There are always initialized here.